reo/nested_inputs.py

```diff
@@ -744,9 +947,9 @@ def list_of_int(input):
         },
         "pbi_max_us_dollars": {
           "type": "float",
-          "min": 0,
-          "max": 1e9,
-          "default": 1e9,
+          "min": 0.0,
+          "max": 1.0e9,
```

Comment:
@Bill-Becker should use max_big_number instead here (and throughout this file in similar cases) for consistency?